src/Symfony/Component/DependencyInjection/Loader/Configurator/EnvConfigurator.php

```diff
@@ -31,6 +31,18 @@ public function __toString()
         return '%env('.implode(':', $this->stack).')%';
     }
 
+    /**
+     * @return $this
+     */
+    public function __call(string $name, array $arguments): self
+    {
+        $processor = strtolower(preg_replace(['/([A-Z]+)([A-Z][a-z])/', '/([a-z\d])([A-Z])/'], '\1_\2', $name));
```

Comment:
We could add a dependency on the string component instead here but not sure if it is worth it.